#!/usr/bin/env python

PACKAGE = 'coral_usb'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('score_thresh', double_t, 0, 'Threshold for confidence score', 0.01, 0.0, 1.0)
gen.add('top_k', int_t, 0, 'Threshold for confidence score', 100, 0, 1000)

exit(gen.generate(PACKAGE, PACKAGE, 'EdgeTPUObjectDetector'))
